code/modules/mob/mob.dm

```diff
@@ -5,11 +5,15 @@
 
 /mob/Destroy() // This makes sure that mobs with clients/keys are not just deleted from the game.
 	unset_machine()
+	qdel(hud_used)
```

Comment:
My next pull is half finished but it gives support for qdel datums, so I'm just going to leave it for now since it's already been there for months.